### PR TITLE
Add more xmx

### PR DESCRIPTION
**Pull request by sharms**
In order to address maximum CPU usage and OutOfMemory errors, we should increase the instance types while we investigate possible causes for increased memory usage.

### CODE REVIEW

File: cf-resource-pools.yml [hunk 1]
```diff
@@ -103,5 +115,5 @@ resource_pools:
   - name: xlarge_errand
     network: cf1
     stemcell: (( merge || meta.stemcell ))
-    cloud_properties: (( merge || resource_pools.large_z1.cloud_properties ))
+    cloud_properties: (( merge || resource_pools.xlarge_z1.cloud_properties ))
```
Comment (pburkholder) [inline cf-resource-pools.yml]: This looks to me like you're correcting a typo from an earlier commit; I don't see how changing xlarge_errand is related to changing the resources for UAA.

File: cf-properties.yml [hunk 2]
```diff
@@ -291,7 +291,7 @@ properties:
     restricted_ips_regex: 10\.\d{1,3}\.\d{1,3}\.\d{1,3}|192\.168\.\d{1,3}\.\d{1,3}|169\.254\.\d{1,3}\.\d{1,3}|127\.\d{1,3}\.\d{1,3}\.\d{1,3}|172\.1[6-9]{1}\.\d{1,3}\.\d{1,3}|172\.2[0-9]{1}\.\d{1,3}\.\d{1,3}|172\.3[0-1]{1}\.\d{1,3}\.\d{1,3}
 
   uaa:
-    catalina_opts: "-Xmx2048m -XX:MaxMetaspaceSize=384m"
+    catalina_opts: "-Xmx4096m -XX:MaxMetaspaceSize=384m"
```
Comment (pburkholder) [inline cf-properties.yml]: How about we jump to 8192?

**Comment by jmcarp** [comment]
Let's hold off on this until we roll out the uaa version without the memory leak.

File: cf-resource-pools.yml (hunk 1 above)
Comment (sharms) [reply cf-resource-pools.yml]: Right - we shouldn't call a xlarge errand with a large size type

**Comment by sharms** [comment]
Remove xmx / UAA instance size changes, but keeping the xlarge_z1/2 types and fixing errand typo

**Pull request closed without merging**